src/NzbDrone.Core/Notifications/Pushsafer/PushsaferSettings.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.Pushsafer
+{
+    public class PushsaferSettingsValidator : AbstractValidator<PushsaferSettings>
+    {
+        public PushsaferSettingsValidator()
+        {
+            RuleFor(c => c.ApiKey).NotEmpty();
+        }
+    }
+
+    public class PushsaferSettings : IProviderConfig
+    {
+        private static readonly PushsaferSettingsValidator Validator = new PushsaferSettingsValidator();
+
+        [FieldDefinition(0, Label = "API Key", HelpLink = "https://www.pushsafer.com/dashboard")]
+        public string ApiKey { get; set; }
+
+        [FieldDefinition(1, Label = "Sound", Type = FieldType.Textbox, HelpText = "Notification sound, leave blank to use the default, set 0 for silent, set 1-50 for custom", HelpLink = "https://www.pushsafer.com/en/pushapi")]
+        public string Sound { get; set; }
```

Comment:
Sounds like it's supposed to be a nullable integer not a string. Same for Icon.

Given that they have special meaning, it might even be better to do a FieldType.Select

Reply:
I will change it to Select

Reply:
Should I list out all the icon options and sound names using Filetype.Select? Its tedious and the API owner can always change them or just add new ones thus we will have to keep it updated all the time. But I can do it if that is the best solution for the UI.

Reply:
I don't think a select is the best idea unless we can programatically get those and that's not something we currently do anywhere else. Personally I'd be okay with a simple text entry as long as we can point people at an official list somewhere.

Reply:
The list is in the HelpText. So I will leave this unchanged